cmake/GNUInstallDirs.cmake

```diff
@@ -0,0 +1,188 @@
+# - Define GNU standard installation directories
```

Comment:
This file is also part of [CMake modules](https://cmake.org/cmake/help/latest/manual/cmake-modules.7.html), why do you need a copy here ?